### PR TITLE
Adding part of the story and creating a new storyline.

### PR DESCRIPTION
**Pull request by ericinlinux**
Adding a bit more to the story

### CODE REVIEW

**Comment by ericinlinux** [comment]
Please review this, @casey-collab

File: portugues/fada/resposta/universo.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Você pergunta qual a resposta para o problema da vida, do universo e tudo mais. Ela então te entrega O Guia do Mochileiro das Galáxias e desaparece. Você então
+
+Chora e joga o livro no lixo.
+
+Abraça sua toalha para secar suas lágrimas e começa a ler o livro.
+
+Desiste de consertar o seu bug e começa a construir um computador capaz de responder a questão da vida do universo e tudo mais pra você.
```
Comment (casey-collab) [inline portugues/fada/resposta/universo.md]: This line is also too long.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ericinlinux** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :sunny:

**Pull request merged**